docs/source/alfacase/case_description.quick_start.rst

```diff
@@ -3,5 +3,84 @@
 Quick Start
 ===========
 
+In this section, it is shown how to work with a ``CaseDescription`` and how to convert an :program:`ALFAcase`
+from an existent project (exported from ALFAsim) to a ``Description``.
 
-In progress
+CaseDescription
+---------------
+
+The ``CaseDescription`` is a Python class that helps users to create a valid ``.alfacase`` file programmatically, by instantiating a
+:class:`CaseDescription <alfasim_sdk.CaseDescription>` object.
+
+To use create a ``CaseDescription`` instance it is necessary to have a Python Interpreter with at least version 3.6 and the
+|sdk| Python package.
+
+
+.. note::
+
+    For more details on how to install Python `check the official docs <https://www.python.org/downloads/>`_.
+    It is recommended that you install the |sdk| using a Python Virtual Environment.
+    For more details, see `Virtual Environments and Packages in Python documentation <https://docs.python.org/3/tutorial/venv.html>`_.
+
+
+From a terminal, and inside a virtual environment, update pip:
+
+>>> python -m pip install -U pip
+
+Install the |sdk| with:
+
+>>> pip install alfasim-sdk
+
+.. note::
+
+    All classes and constants that will be used to configure a project can be accessed through the main module of ``alfasim_sdk``
+
+
+In section :ref:`alfacase-reference-section` it is possible to check all available classes and options,
+as well which types and validators are applied at the time of object instantiation.
+
+To know the type is important to create instantiate a class properly, for example, if you try to create a
+:class:`NodeDescription <alfasim_sdk.NodeDescription>`, the ``node_type`` attribute
+must be a string one of the options available on :class:`NodeCellType <alfasim_sdk.NodeCellType>`, otherwise, a
+``ValueError`` is raised.
+
+.. code-block:: python
+
+    >>> import alfasim_sdk
+    >>> alfasim_sdk.NodeDescription(name=1, node_type=alfasim_sdk.NodeCellType.Pressure)
+    [ ... ]
+    ValueError: 'node_type' must be in <enum 'NodeCellType'> (got 1)
+
+
+As already informed at the beginning of the section, ALFAsim accepts only ``.alfacase`` files, the description classes
+are only tools that help the creation and manipulation of a project programmatically in an easier way.
+
+To convert a ``Description`` to alfacase you can use the function ``convert_description_to_alfacase`` and to convert a
```

Comment:
I just noticed the names of the functions are not equivalent, one says `description` and the other `case`. I'm too much familiar with referencing CaseDescription as both case and description, but maybe it is confusing for users?

Reply:
Which one we should stick with?
After writing these docs I kind of liked to refer it as `Descriptions`

Reply:
Yep, _case_ may look like it is referring to the alfacase files.